app/repository/hbase/localunit/HBaseRestLocalUnitRepository.scala

```diff
@@ -25,16 +26,32 @@ class HBaseRestLocalUnitRepository @Inject() (
 
   override def retrieveLocalUnit(ern: Ern, period: Period, lurn: Lurn): Future[Either[ErrorMessage, Option[LocalUnit]]] = {
     logger.info(s"Retrieving Local Unit with [$ern] [$lurn] for [$period].")
-    restRepository.findRow(config.tableName, LocalUnitRowKey(ern, period, lurn), DefaultColumnGroup).map(fromErrorOrRow)
+    restRepository.findRow(config.tableName, LocalUnitQuery.byRowKey(ern, period, lurn), DefaultColumnGroup).map(fromErrorOrRow)
```

Comment:
Still not too convinced with DefaultColumnGroup. The DB conventional name for column group name is ColumnFamily - maybe rename its to 'DefaultColumnFamily' to keep the naming consistent from DB to client